src/markd/renderer.cr

```diff
@@ -14,7 +14,8 @@ module Markd
       @last_output = string
     end
 
-    def cr
+    # render a Line Feed character
+    def lf
```

Comment:
If this is changed, I'd suggest to make it an even more expressive name like `newline`.